db/blob/blob_source_test.cc

```diff
@@ -525,26 +643,52 @@ TEST_F(BlobSourceTest, MultiGetBlobsFromCache) {
 
     read_options.fill_cache = true;
     read_options.read_tier = ReadTier::kReadAllTier;
+    get_perf_context()->Reset();
+    statistics->Reset().PermitUncheckedError();
 
     // Get half of blobs
     blob_source.MultiGetBlob(read_options, key_refs, blob_file_number,
                              file_size, offsets, sizes, statuses, values,
                              &bytes_read);
 
+    uint64_t fs_read_bytes = 0;
+    uint64_t ca_read_bytes = 0;
     for (size_t i = 0; i < num_blobs; ++i) {
       if (i % 2 == 0) {
         ASSERT_OK(statuses_buf[i]);
         ASSERT_EQ(value_buf[i], blobs[i]);
         ASSERT_TRUE(blob_source.TEST_BlobInCache(blob_file_number, file_size,
                                                  blob_offsets[i]));
+        ca_read_bytes += blob_sizes[i];
       } else {
         statuses_buf[i].PermitUncheckedError();
         ASSERT_TRUE(value_buf[i].empty());
         ASSERT_FALSE(blob_source.TEST_BlobInCache(blob_file_number, file_size,
                                                   blob_offsets[i]));
+        fs_read_bytes +=
+            blob_sizes[i] + keys[i].size() + BlobLogRecord::kHeaderSize;
```

Comment:
This is kind of confusing... doesn't this really belong in the other branch too? It is the even-numbered blobs that got read from storage by `MultiGetBlob`, no?

Reply:
You're correct! Only the other branch should contain these lines. The blobs with odd numbers have not yet been read.